mailbox/plugin/quota-mailing-memory/src/test/java/org/apache/james/eventsourcing/InMemoryEventStoreTest.java

```diff
@@ -1,11 +1,9 @@
 package org.apache.james.eventsourcing;
 
-import org.apache.james.eventsource.InMemoryEventStore;
+import org.apache.james.mailbox.quota.memory.InMemoryEventStoreExtension;
+import org.junit.jupiter.api.extension.ExtendWith;
 
+@ExtendWith(InMemoryEventStoreExtension.class)
```

Comment:
why do you think it's better ?

Reply:
Well... Because that is what we do in JUNIT 5 code, that the extension already exist, etc...

I must admit I don't understand your question....